apps/backoffice-v2/src/common/enums.ts

```diff
@@ -1,5 +1,14 @@
 import { TObjectValues } from './types';
 
+export const BroadcastChannels = {
+  OPEN_DOCUMENT_IN_NEW_TAB: 'OPEN_DOCUMENT_IN_NEW_TAB',
+};
+
+export enum BroadcastChannelEvents {
+  'OPEN_DOCUMENT_IN_NEW_TAB',
+  'OPEN_DOCUMENT_IN_NEW_TAB_ACK',
+}
```

Comment:
Remember native TypeScript enums are ill advised. https://www.youtube.com/watch?v=jjMbPt_H3RQ
```suggestion
export const BroadcastChannel = {
  OPEN_DOCUMENT_IN_NEW_TAB: 'OPEN_DOCUMENT_IN_NEW_TAB',
} as const;

export const BroadcastChannelEvent {
  OPEN_DOCUMENT_IN_NEW_TAB: 'OPEN_DOCUMENT_IN_NEW_TAB',
  OPEN_DOCUMENT_IN_NEW_TAB_ACK: 'OPEN_DOCUMENT_IN_NEW_TAB_ACK',
} as const;
```